connectors/migrations/db/migration_20.sql

```diff
@@ -0,0 +1,3 @@
+-- Migration created on Sep 27, 2024
+ALTER TABLE "public"."slack_bot_whitelist" ADD COLUMN "botId" VARCHAR(255);
+ALTER TABLE "slack_bot_whitelist" ALTER COLUMN "botName" DROP NOT NULL;ALTER TABLE "slack_bot_whitelist" ALTER COLUMN "botName" DROP DEFAULT;ALTER TABLE "slack_bot_whitelist" ALTER COLUMN "botName" TYPE VARCHAR(255);
```

Comment:
Why are we changing the type of the `botName`column ?

Also, can we format the file and use consistent style (either always fully qualified table name or never)